### PR TITLE
[REVIEW] Add new dependencies for Arrow / Dask

### PR DESCRIPTION
**Pull request by kkraus14**
Changes incoming in cuDF that will need these changes, but please do not merge before then.

See: https://github.com/rapidsai/cudf/pull/2353

### CODE REVIEW

**Comment by dillon-cullinan** [comment]
@mike-wendt I talked with Keith about moving a lot of the installs in cuDF's gpu build into the gpuCI environment so it stops cluttering things and taking more time.

Was hoping you could chime in on what standards we should set on which packages go into the build script and which ones should go into the environment.

File: Dockerfile.centos7 [hunk 1]
```diff
@@ -33,7 +32,13 @@ ARG NVIDIA_CONDA_LABEL=nvidia/label/cuda${CUDA_MAJORMINOR_VERSION}
 ARG RAPIDSAI_CONDA_LABEL=rapidsai/label/cuda${CUDA_MAJORMINOR_VERSION}
 ARG RMM_VERSION=0.7.*
 ARG PANDAS_VERSION=0.23.4
```
Comment (galipremsagar) [inline Dockerfile.centos7]:
```suggestion
ARG PANDAS_VERSION=0.24.2
```
This might also have to be changed.

**Comment by mike-wendt** [comment]
@kkraus14 what is the forecast for closing the above PR? We are in the middle of moving to a different set of base containers at the moment. Trying to figure out what the best approach will be to migrate everyone to the new base containers and get your changes in the right place.

**Comment by kkraus14** [comment]
@mike-wendt this is ready for review now as the above PR has been merged.

**Pull request merged**